src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,863 @@
+            <p><strong>Default value</strong></p>
+            <p>(host-specific default)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>preferred<wbr />DockedSize</inlineCode></td>
+        <td><inlineCode>{'{'} width: number, height: number {'}'}</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>The preferred size of the view when docked, if it can be docked, or for modal dialogs that have no reference_node_id. The host may not be able to guarantee this size.</p>
+            <p><strong>Default value</strong></p>
+            <p>(host-specific default)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>preferred<wbr />FloatingSize</inlineCode></td>
+        <td><inlineCode>{'{'} width: number, height: number {'}'}</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>The preferred size of the panel when floating. The host may not be able to guarantee this size. See minimumSize for the format.</p>
+            <p><strong>Default value</strong></p>
+            <p>(host-specific default)</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### HostDefinition
+
+<p>UXP supports a number of different host applications. The host definition specifies which host app the plugin supports.</p>
+<h4>Properties</h4>
+<table >
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><inlineCode>app</inlineCode> *</td>
+        <td><inlineCode>"XD" <wbr />| "PS" <wbr />| "ID"</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The host app that the plugin supports.</p>
+            <p>Possible values:</p>
+            <ul>
+                <li><inlineCode>XD</inlineCode>: Adobe XD</li>
+                <li><inlineCode>PS</inlineCode>: Adobe Photoshop</li>
+                <li><inlineCode>ID</inlineCode>: Adobe InDesign</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>minVersion</inlineCode> *</td>
+        <td><inlineCode>string</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The minimum version of the host app that the plugin supports. The version string follows the format of <inlineCode>x.y.z</inlineCode>.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>maxVersion</inlineCode></td>
+        <td><inlineCode>string</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>The maximum version of the host app that the plugin supports.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (the latest version of the host app)</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### PermissionsDefinition
```

Comment:
Missing documentation for "allowCodeGenerationFromStrings" and "ipc" -> enablePluginCommunication. I have recipes for these. You can use the following
- https://github.com/AdobeDocs/uxp-indesign/blob/6d1ad4ff1c2e096d01edaf91809cd8b4c8f4cad7/src/pages/resources/recipes/html-events.md
- https://github.com/AdobeDocs/uxp-indesign/blob/6d1ad4ff1c2e096d01edaf91809cd8b4c8f4cad7/src/pages/resources/recipes/ipc.md

Reply:
Added.